int64 state
int64 penalty
int64 firstHalf
int64 kickoffTeam
